apps/admin/frontend/src/components/export_election_package_modal_button.tsx

```diff
@@ -50,13 +46,16 @@ export function ExportElectionPackageModalButton(): JSX.Element {
     setSaveState({ state: 'unsaved' });
   }
 
-  async function saveElectionPackageToUsb() {
-    const result = await saveElectionPackageToUsbMutation.mutateAsync();
-    if (result.isErr()) {
-      setSaveState({ state: 'error', error: result.err() });
-      return;
-    }
-    setSaveState({ state: 'saved' });
+  function saveElectionPackage() {
+    saveElectionPackageToUsbMutation.mutate(undefined, {
+      onSuccess: (result) => {
+        if (result.isErr()) {
+          setSaveState({ state: 'error', error: result.err() });
+        } else {
+          setSaveState({ state: 'saved' });
+        }
+      },
+    });
```

Comment:
@votingworks/vxsuite-eng Just a reminder that if you use `.mutateAsync` without a `try/catch`, any exceptions will cause an unhandled promise rejection, which makes tests crash in weird ways.

Recommended ways to call a mutation are:
```ts
// If you don't care about waiting for completion
mutation.mutate(args)

// If you need to wait for completion or results
mutation.mutate(args, { 
  onSuccess: (results) => {
     ... do the next thing ... 
  }
})

// If you need to pass an async event handler to another 
// component, or using a promise will make things
// way more  readable (e.g. multiple mutations in a row)
async function onSomeEvent() {
  try {
    await mutation.mutateAsync(args)
  } catch (e) {
    // Handled by query client default error handling
   }
}
```